lib/tasks/seek_upgrades.rake

```diff
@@ -235,6 +236,40 @@ namespace :seek do
     end
   end
 
+  task(add_policies_to_existing_sample_types: [:environment]) do
```

Comment:
I think generally this needs a rethink and currently doesn't reflect the original permissions.

Would need checking more closely, but generally:
* make visible if linked to public samples, otherwise hidden
* add visible permissions for each project
* give manage rights to the project administrators